### ISSUE TITLE
Where is the link for NLP models

### ISSUE THREAD

**Issue by yqtianust** [openvinotoolkit/nncf#611]
Hi,

I cannot find any documentation introducing the NLP models except the table in Readme. Could anyone please tell me where to find the pre-trained models and whether any sample code to show how to use them with NNCF?

Thanks for the help.

**Comment by vshampor** [comment]
Greetings, @yqtianust !

The results for the NLP models in the README were obtained using third-party training pipelines - namely, Huggingface Transformers. The instructions on how to integrate NNCF into transformers (although for a quite outdated transformers repo state) can be found in the README.md at https://github.com/openvinotoolkit/nncf/tree/develop/third_party_integration/huggingface_transformers

For such models, we do not currently distribute checkpoints and instead only provide the NNCF config files and the sample command lines to be used with the third-party training pipelines - basically recipes for users to obtain their own checkpoints which hopefully give close results to what is claimed in the READMEs.

**Comment by yqtianust** [comment]
Thanks. I will have a try.

**Comment by masahi** [comment]
Hi @vshampor, is there any plan to publish fine-tuned transformer model checkpoints? My GPU may not be powerful enough to run full QAT on my end.

**Comment by masahi** [comment]
Thanks @vshampor. Following the instruction, I was able to generate int8 `DistilBERT-SST-2` and `BERT-XNLI` is currently cooking. I understand the policy. 

Indeed, I'm interested in `bert-large` int8, but the OMZ link https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/bert-large-uncased-whole-word-masking-squad-int8-0001 only contains OpenVINO representation (*.xml and *.bin files), not ONNX. Let me know if there is any chance if you could publish the ONNX file as well.
